bake_common.py

```diff
@@ -1168,8 +1168,41 @@ def bake_to_vcol(mat, node, root_ch, extra_channel=None, extra_multiplier=1.0):
 
     mat.node_tree.links.new(rgb, emit.inputs[0])
 
+    # To avoid duplicate code, define the function here
+    def bake_alpha_to_vcol():
+        origin_vcol_name = ''
+        for obj in objs:
+            # Gets the origin vertex color of the objs, Their names are normally the same
+            origin_vcol_name = get_active_vertex_color(obj).name
```

Comment:
I think you should pass bake vcol name to bake_to_vcol function and try to get the vcol based on that name. I think it's safer that way.